scripts/check-trans.php

```diff
@@ -34,7 +32,7 @@
   Usage:
   <?php echo $argv[0]; ?> <language-code>
 
-  Authors:Yannick Torr�s <yannick@php.net>
+  Authors:Yannick Torr�s <yannick@php.net>
```

Comment:
What happened here? Editor encoding issues? There are a few more instances in this script where similar changes happened.